build.gradle

```diff
@@ -55,6 +55,17 @@ buildscript {
     }
 }
 
+buildscript {
```

Comment:
This is the wrong place to add this section (or the new plugin). Also: this seems to be the older way to add a Gradle plugin... which makes me a bit suspicious. When we upgrade to Gradle 8 (which is already out for a while) then these old plugins are very likely to break.
Also: not sure if lower-/uppercase for log level etc. is really an issue here? If not required to change then please leave as is to keep the PR as minimal as possible.